src/js/components/StarRating/__tests__/StarRating-tests.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
Can we make this a typescript file? Trying to make sure any new test files that are added are typescript

Reply:
done

Reply:
Looks like this got switched back to a `.js` file